org.seul.pingus.json

```diff
@@ -14,7 +14,17 @@
         "--socket=pulseaudio"
     ],
     "modules": [
-        "shared-modules/sigc++/sigc++-2.0.json",
+        {
+        	"name": "libsigc++-2.0",
```

Comment:
We could probably update the other modules (including shared-modules) as well as a part of this PR.